setup.cfg

```diff
@@ -0,0 +1,27 @@
+[bdist_wheel]
+universal=1
+
+[metadata]
+name = nmigen
+author = whitequark
+author_email = whitequark@whitequark.org
+description = Python toolbox for building complex digital hardware
+license = BSD
+project_urls =
+    Source Code=https://github.com/nmigen/nmigen
+    Bug Tracker=https://github.com/nmigen/nmigen/issues
+
+[options]
+python_requires = ~=3.6
+install_requires =
+    pyvcd~=0.2.0
```

Comment:
Please keep the comments, they're useful to explain the scope where the dependency is required.